src/hotspot/os/linux/os_linux.cpp

```diff
@@ -988,6 +989,15 @@ bool os::create_thread(Thread* thread, ThreadType thr_type,
       return false;
     }
 
+    {
+      // Print current timer slack, if available
+      int slack = prctl(PR_GET_TIMERSLACK);
+      if (slack >= 0) {
+        log_info(os, thread)("Thread \"%s\" (pthread id: " UINTX_FORMAT ") timer slack: %dns",
+                             thread->name(), (uintx) tid, slack);
+      }
+    }
+
```

Comment:
Maybe also in create_attached_thread? And then it would make sense to factor this out into a small static helper.

Reply:
As discussed in other thread, I'd prefer to keep the attached threads out of the picture here.